website/static/templates.json

```
@@ -2928,6 +2928,25 @@
       "new"
     ]
   },
+  {
+    "title": "Data API builder Quickstart - Azure SQL",
+    "description": "A quick application to get started using Data API builder to create a REST API for Azure SQL.",
+    "preview": "./templates/images/dab-azure-sql-quickstart.png",
+    "website": "https://github.com/azure-samples",
```

```suggestion
    "website": "https://learn.microsoft.com/azure/data-api-builder/quickstart-azure-sql",
```

Per https://github.com/Azure/awesome-azd/pull/471#discussion_r1934519677